graph/bolt/quadstore.go

```diff
@@ -184,7 +185,7 @@ var (
 	metaBucket = []byte("meta")
 )
 
-func (qs *QuadStore) ApplyDeltas(deltas []graph.Delta) error {
+func (qs *QuadStore) ApplyDeltas(deltas []graph.Delta, ignoreDuplicate bool, ignoreMissing bool) error {
```

Comment:
Delete first bool type. Also below.

The length of this signature troubles me for such a simple operation.